h/static/scripts/directive/publish-annotation-btn.js

```diff
@@ -47,7 +9,23 @@ function PublishAnnotationBtnController($scope) {
 // @ngInject
 module.exports = function () {
   return {
-    controller: PublishAnnotationBtnController,
+    bindToController: true,
```

Comment:
This is a new feature in Angular 1.3 which makes properties in a directive's isolate scope accessible on the controller.